bundles/storage/org.eclipse.smarthome.storage.json/src/main/java/org/eclipse/smarthome/storage/json/internal/JsonStorage.java

```diff
@@ -21,6 +21,8 @@
 import java.util.concurrent.ConcurrentHashMap;
 
 import org.eclipse.jdt.annotation.NonNull;
```

Comment:
There is still @NonNull in use in this class, which should be superseded by @NonNullByDefault.